app/views/home/_location_dropdown.html.erb

```diff
@@ -1,9 +0,0 @@
-<li class="dropdown-header"><%= _('Location') %></li>
-<li><%= link_to(_('Any Location'), main_app.clear_locations_path) %></li>
-<% if User.current.allowed_to?(:view_locations) %>
```

Comment:
I think that feature is missed? you probably need to pass if a prop if a user have permissions to view_locations or view_orgs